src/mame/sinclair/specnext_sprites.cpp

```diff
@@ -0,0 +1,282 @@
+// license:BSD-3-Clause
+/**********************************************************************
+
+    Spectrum Next Sprites
+
+TODO:
+- Relative Sprite, Composite Type
+- Secondary Palette
+
+**********************************************************************/
+
+#include "emu.h"
+#include "screen.h"
+
+#include "specnext_sprites.h"
```

Comment:
The first thing you `#include` after the prefix header should be the module header.